src/Sylius/Bundle/ThemeBundle/Translation/DependencyInjection/Compiler/TranslatorFallbackLocalesPass.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ThemeBundle\Translation\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+final class TranslatorFallbackLocalesPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        try {
+            $symfonyTranslator = $container->findDefinition('translator.default');
```

Comment:
If you need that, why catching exception at all? Should we silence the exception?